rancid-tomatillos/src/Card/PreviewCard.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 function PreviewCard({id, poster_path, title, average_rating}) {
     return (
         <div itemID={id} className="preview-card">
-            <img src={poster_path}></img>
+            <img src={poster_path} alt={`poster of ${title}`}></img>
```

Comment:
Think this was a good call - allows us to check for image in our test file as well as have alt text in case of missing/incorect image hosting and for accessibility